product_currency/README.rst

```diff
@@ -1,13 +1,20 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+.. |company| replace:: ADHOC SA
+
+.. |company_logo| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-logo.png
+   :alt: ADHOC SA
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
    :alt: License: AGPL-3
 
 ================
 Product Currency
 ================
 
-Allow to define on products a different currency from the default one (related to product company)
-
+This module allow to define on products a different currency from the default one (related to product company).
```

Comment:
> from the default one (related to product company).

I think is confusing. I will suggest something like:

from the default one (which is company currency were this product is defined)